app/containers/ActionSheet/BottomSheetContent.tsx

```diff
@@ -0,0 +1,58 @@
+import { Text } from 'react-native';
+import React from 'react';
+import { BottomSheetView, BottomSheetFlatList } from '@gorhom/bottom-sheet';
+
+import { Button } from './Button';
+import I18n from '../../i18n';
+import { useTheme } from '../../theme';
+import { IActionSheetItem, Item } from './Item';
+import { TActionSheetOptionsItem } from './Provider';
+import styles from './styles';
+import * as List from '../List';
+
+interface BottomSheetContentProps {
+	hasCancel?: boolean;
+	options?: TActionSheetOptionsItem[];
+	hide: () => void;
+	children?: React.ReactElement | null;
+}
+
+const BottomSheetContent = React.memo(({ options, hasCancel, hide, children }: BottomSheetContentProps) => {
+	const { theme, colors } = useTheme();
+
+	const renderFooter = () =>
+		hasCancel ? (
+			<Button
+				onPress={hide}
+				style={[styles.button, { backgroundColor: colors.auxiliaryBackground }]}
+				// TODO: Remove when migrate Touch
+				theme={theme}
+				accessibilityLabel={I18n.t('Cancel')}>
+				<Text style={[styles.text, { color: colors.bodyText }]}>{I18n.t('Cancel')}</Text>
+			</Button>
+		) : null;
+
+	const renderItem = ({ item }: { item: IActionSheetItem['item'] }) => <Item item={item} hide={hide} />;
+
+	if (options) {
+		return (
+			<BottomSheetFlatList
```

Comment:
The expected UX is not to scroll on the FlatList, but to move the whole action sheet.
On this video, you can see it's scrolling if I touch inside and I need to scroll from the handle to actually scroll the action sheet.

https://user-images.githubusercontent.com/804994/168338394-0b3ae8b2-56e0-4bbe-a936-54074d23c0ca.mp4